Example/Tests/Features/NativeFeatures/native_example_order.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Simple Feature File Ordering
+
+    Scenario: B This is a very simple example of ordering
+        Given this should be executed before
```

Comment:
Any chance you can add examples to this scenario - just to get the test coverage up?

```
    Scenario: B This is a very simple example of ordering
        Given this should be executed before A with example value <value>

        Examples:
        | value |
        | 1 |
        | 2 |
        | 3 |
```

(That would also mean making `hasExecutedBeforeStep` a count instead of a bool I guess?)